### PR TITLE
[PartEditor] Introduce makeDefaultPartiton method

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce Partition class and makeDefaultPartiton method
for default 'partition' section.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #641, from draft #647

File: src/PartEditor/PartEditor.ts [hunk 1]
```diff
@@ -126,6 +132,21 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider {
...
 
+  private makeDefaultPartiton() {
+    let partition: Partition = {};
+    let backends = '';
+    for (let idx = 1; idx < this._backEndNames.length; idx++) {
+      if (backends.length) {
+        backends = backends + ',';
+      }
+      backends = backends + this._backEndNames[idx];
+    };
```
Comment (mhs4670go) [inline src/PartEditor/PartEditor.ts:143]: How about this one?
```suggestion
    let backends = this._backEndNames.join(",");
    };
```

Comment (seanshpark) [reply src/PartEditor/PartEditor.ts:143]: `let idx = 1;` starts from second item.
is there something like copy array from second item?

Comment (mhs4670go) [reply src/PartEditor/PartEditor.ts:143]: Oh, my bad. Then, how about this one?
```typescript
let backends = this._backEndNames.slice(1, this._backEndNames.length).join(',');
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**